src/components/shared/AddTextInput.tsx

```diff
@@ -0,0 +1,37 @@
+import { AddCircleOutline } from "@mui/icons-material";
```

Comment:
This file is not clear. The title is quite generic, but looking at the content I see LoRaWAN stuff.

Reply:
It's generic as it's used in the Lorawan actions, it's a text field and I use it for all LoraWAN options

Reply:
So let's find a better name, reflecting the LoRaWAN stuff. e.g. AddLoRaWANTextInput